lib/parser/models.ex

```diff
@@ -221,6 +232,29 @@ end
       end
     end
 
+    defmodule TEnumValue do
+      @moduledoc """
+      A reference to an enum value
+      For example, in a constant or default value.
+
+         const string DEFAULT_WEATHER = Weather.SUNNY;
+     """
+      @type t :: %TEnumValue{enum_name: atom, enum_value: atom, type: atom}
+      defstruct enum_name: nil, enum_value: nil, type: nil
```

Comment:
I did this intentionally; in the binary protocol we make a lot of assumptions about what name and value mean, an din this case, we can't just do the generic thing (resolve the struct name) and replace the original with the resolution. I'm trying to be clear that we're talking about a window into an enum.